x-pack/plugins/siem/server/lib/timeline/saved_object.ts

```diff
@@ -98,13 +99,23 @@ export const getTimelineByTemplateTimelineId = async (
 
 /** The filter here is able to handle the legacy data,
  * which has no timelineType exists in the savedObject */
-const getTimelineTypeFilter = (timelineType: string | null) => {
-  return timelineType === TimelineType.template
-    ? `siem-ui-timeline.attributes.timelineType: ${TimelineType.template}` /** Show only whose timelineType exists and equals to "template" */
-    : /** Show me every timeline whose timelineType is not "template".
-       * which includes timelineType === 'default' and
-       * those timelineType doesn't exists */
-      `not siem-ui-timeline.attributes.timelineType: ${TimelineType.template} and not siem-ui-timeline.attributes.timelineType: ${TimelineType.draft}`;
+const getTimelineTypeFilter = (timelineType: string | null, includeDraft: boolean) => {
+  const typeFilter =
+    timelineType === TimelineType.template
+      ? `siem-ui-timeline.attributes.timelineType: ${TimelineType.template}` /** Show only whose timelineType exists and equals to "template" */
+      : /** Show me every timeline whose timelineType is not "template".
+         * which includes timelineType === 'default' and
+         * those timelineType doesn't exists */
+        `not siem-ui-timeline.attributes.timelineType: ${TimelineType.template}`;
+
+  /** Show me every timeline whose status is not "draft".
+   * which includes status === 'active' and
+   * those status doesn't exists */
+  const draftFilter = includeDraft
+    ? `siem-ui-timeline.attributes.status: draft`
```

Comment:
please consider use TimelineStatus.draft

Reply:
Fixed, Thank you @angorayc